### PR TITLE
Remove '.yml' from the computers/codes setting fields.

### PR DESCRIPTION
**Pull request by yakutovicha**

### CODE REVIEW

**Review by sphuber — COMMENTED** [review]
Using the `[:-4]` syntax is quite fragile. Wouldn't it be better to define the suffixes that are allowed and explicitly remove those if defined. i.e. currently `.yaml` would fail, but that is a common extension for YAML files is it not?

File: .make_ghpages/generate_json.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 # Extract all the data. 
 
+ALLOWED_YAML_SUFFIXES = ['.yml', '.yaml']
```
Comment (ltalirz) [inline .make_ghpages/generate_json.py:13]: Aren't we shooting ourselves in the foot by allowing both suffixes in the registry?
We make our life harder both for users and maintainers.

I would propose to add a stupid pre-commit hook that raises whenever someone tries to commit a `.yaml` file, asking them to use `.yml` instead.

What do you think?

File: .make_ghpages/generate_json.py [hunk 2]
```diff
@@ -19,9 +21,16 @@
...
-        		final_dict[domain][computer][configuration] = yaml.load(yaml_file, Loader=yaml.FullLoader)
+
+        for configuration in [ conf for conf in os.listdir(library_path/domain/computer) if conf != "README"]:
+            with open(library_path/domain/computer/configuration) as yaml_file:
+                for suffix in ALLOWED_YAML_SUFFIXES:
+                    if configuration.endswith(suffix):
+                        configuration = configuration[:len(suffix)]
+                        break
+                else:
+                    raise ValueError(f"The file {configuration} has unsupported extention. Please use one of: {ALLOWED_YAML_SUFFIXES}")
```
Comment (ltalirz) [inline .make_ghpages/generate_json.py:32]:
```suggestion
                    raise ValueError(f"The file {configuration} has unsupported extension. Please use one of: {ALLOWED_YAML_SUFFIXES}")
```

File: .make_ghpages/generate_json.py (hunk 1 above)
Comment (yakutovicha) [reply .make_ghpages/generate_json.py:13]: I am fine with using one of those, but why did we chose `yml` and not `yaml`. According to the [yaml FAQ](https://yaml.org/faq.html) `yaml` is the recommended way.

Comment (ltalirz) [reply .make_ghpages/generate_json.py:13]: Thanks, I wasn't aware. `yaml` it is then!

Comment (yakutovicha) [reply .make_ghpages/generate_json.py]: I renamed all the configuration files.

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks @yakutovicha !

All good, just the README to go

**Comment by ltalirz** [comment]
After updating the README, feel free to merge directly without further review

**Pull request merged**